src/cmd.rs

```diff
@@ -4,6 +4,7 @@ mod changelog;
 mod check;
 mod commit;
 mod version;
+mod config;
```

Comment:
Run `cargo +nightly fmt` it will order the modules and change some formatting here and there

- [ ] `cargo +nightly fmt`